spec/forms/state_file/nj_tenant_rent_paid_form_spec.rb

```diff
@@ -62,6 +51,28 @@
         end
       end
     end
+
+    context "valid params" do
+      context "can be a decimal" do
+        let(:params) do
+          { :rent_paid => "123.45" }
+        end
+
+        it "is valid" do
+          expect(form.valid?).to eq true
+        end
+      end
+
+      context "can be an integer" do
```

Comment:
[dust] It feels a little redundant to write out these identical tests each time we use the money field. Maybe this is a place to use `shared_example` and `it_behaves_like`? https://stackoverflow.com/questions/30104215/how-to-reuse-the-same-test-multiple-times-in-rspec

Reply:
moved to separate ticket https://github.com/newjersey/affordability-pm/issues/147